P2/task_5.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Created by yahor.turets on 20.07.2017.
+ */
```

Comment:
Object sorter is missed. According to task you need to provide object with sorting options.